plugins/in_node_exporter_metrics/ne_utils.c

```diff
@@ -178,6 +178,79 @@ int ne_utils_file_read_lines(const char *mount, const char *path, struct mk_list
     return 0;
 }
 
+/*
+ * Read a file and every non-empty line is stored as a flb_slist_entry in the
+ * given list.
+ */
+int ne_utils_file_read_sds(const char *mount, 
```

Comment:
Rename the variables with meaningful names.

Reply:
Add the missing safety checks for  `flb_sds_cat_safe`

Reply:
The names of the arguments follow the convention used in the other functions in the same file.